test/testpy/__init__.py

```diff
@@ -53,11 +53,15 @@ def AfterRun(self, result):
     # delete the whole tmp dir
     try:
       rmtree(self.tmpdir)
+      if "NODE_PIPE_DIR" in os.environ:
+        rmtree(os.environ["NODE_PIPE_DIR"]);
```

Comment:
Right on Windows it already uses a directory that is outside of the source directory, so we're fine. But on all other supported platforms, we should be able to use `mktemp` or something similar, or am I missing something?